src/pages/index.astro

```diff
@@ -2,64 +2,82 @@
 import BaseHead from '../components/BaseHead.astro';
 import Header from '../components/Header.astro';
 import Footer from '../components/Footer.astro';
-import { SITE_TITLE, SITE_DESCRIPTION } from '../consts';
+import { SITE_TITLE, SITE_DESCRIPTION} from '../consts';
+import {background_config} from '../consts'
+import Particles from "astro-particles";
+import type { ISourceOptions } from "tsparticles-engine";
+
+const options: ISourceOptions = background_config
 ---
 
 <!doctype html>
 <html lang="en">
-	<head>
-		<BaseHead title={SITE_TITLE} description={SITE_DESCRIPTION} />
-		<script src="https://cdnjs.cloudflare.com/ajax/libs/three.js/r128/three.min.js" defer></script>
+    <head>
+        <script type="module">FDE49E
+            import astroParticles from 'https://cdn.jsdelivr.net/npm/astro-particles@2.10.0/+esm'
```

Comment:
Is there a way to add some kind of integrity sha to this import? Better for security ideally.

Reply:
Hey @vr-varad , I don't think this one was addressed?

Reply:
i am not aware of how to do that u can provide some extra info on exactly what u mean it would be helpful.

Reply:
This kind of thing is used to verify that the script you're importing is the one you're expecting from a CDN using a unique identifier. I'm not sure if it'll work with this import syntax, but it's usually used with script tags. Can you investigate a little and see if it can be used here?


https://www.w3schools.com/tags/att_script_integrity.asp

Reply:
Actually, I found the cdn https://www.jsdelivr.com/package/npm/astro-particles here and thought would be helpful but if you follow the official docs https://github.com/tsparticles/astro its not the part of code and could work with out it, so I can remove it if u say.
And I got the point regarding integrity never heard of it before got to know something new thanks but the cdn link is from any random website so didn't contain sha and couldn't find it on cdnjs which provides integrity sha.

Reply:
@sabaimran

Reply:
If you can remove the import from cdn and import it as a normal package, let's do that. 

Otherwise, you should be able to generate the sha programmatically. There may also be online services that do so: https://app.khoj.dev/share/chat/how-can-i-create-an-integrity-sha-for-a-file-in-im/

Reply:
basically, the cdn provider provides sha but the cdn link I got from doesn't provide the sha.
I will remove it. :)